src/pages/membership/components/youthProfileForm/YouthProfileForm.tsx

```diff
@@ -45,30 +41,41 @@ const schema = Yup.object().shape({
   approverFirstName: Yup.string()
     .min(2, 'validation.tooShort')
     .max(255, 'validation.tooLong')
-    .when([], (schema: Yup.StringSchema) => {
-      return AGE < ageConstants.ADULT
+    .when(['birthDate'], (value: string, schema: Yup.StringSchema) => return isConsentRequired(birthDate, schema)
```

Comment:
Hm, this last commit doesn't really help does it. What you're saying is that we can't share the isConsentRequired logic between the different fields in any simple way due to typing issues? 

Then I think we should revert my last commit and merge it as it was before..

Reply:
I think it should work, I'll fix it manually.

Reply:
The helper function is now inside the .when() and propably crashed everything.

Reply:
Seems to be working now 👍

Reply:
Great!